### PR TITLE
Revise CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by jtcohen6**
### Feature request template
* Add `Are you interested in contributing this feature?`

### CONTRIBUTING
* It's mostly really good! I still want to show this to some less-technical colleagues and ask where they'd feel tripped up
* Remove the bit about forks not running all CI tests
* Add more issue labels and descriptions
* Move the "tools" section up higher
* Emphasize that passing unit + Postgres integration tests are most important
* Small copyedit changes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -18,3 +18,6 @@ Is this feature database-specific? Which database(s) is/are relevant? Please inc
 
 ### Who will this benefit?
 What kind of use case will this feature be useful for? Please be specific and provide examples, this will help us prioritize properly.
+
+### Are you interested in contributing this feature?
+Let us know if you want to write some code, and what help you want in getting started.
```
Comment (drewbanin) [inline .github/ISSUE_TEMPLATE/feature_request.md:23]:
```suggestion
Let us know if you want to write some code, and how we can help.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ After creating an issue, a dbt maintainer will follow up with you to explore you
 
 ### Submitting a change
 
-If the issue is appropriately well-scoped and describes a beneficial change to the dbt codebase, then anyone may submit a Pull Request to implement the functionality described in the issue (see the sections below on how to do this).
+If an issue is appropriately well scoped and describes a beneficial change to the dbt codebase, then anyone may submit a Pull Request to implement the functionality described in the issue. See the sections below on how to do this.
+
+The dbt maintainers will add a `good first issue` label if an issue is suitable for a first-time contributor. Often if a required code change is small and self-contained, only applicable to one database adapter, or a net-new addition that does not impact existing functionality. You can see the list of currently open issues on the [Contribute](https://github.com/fishtown-analytics/dbt/contribute) page.
```
Comment (drewbanin) [inline CONTRIBUTING.md:25]: this reads kind of funny.
> Often if a required code change is small and self-contained, only applicable to one database adapter, or a net-new addition that does not impact existing functionality.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -30,13 +38,15 @@ The dbt maintainers use labels to categorize open issues. Some labels indicate t
...
-| good first issue | This issue does not require deep knowledge of the dbt codebase to implement. This issue is appropriate for a first-time contributor to implement. |
-| snoozed | This issue describes a good idea, but one which will probably not be addressed in a six-month time horizon. The dbt maintainers will revist these issues periodically and re-prioritize them accordingly. |
-| triage | This is a new issue which has not yet been reviewed by a dbt maintainer. This label is removed when a maintainer reviews and responds to the issue. |
-| stale | This is an old issue which has not recently been updated. Stale issues will periodically be closed by dbt maintainers, but they can be re-opened if the discussion is restarted. |
-| wontfix | This issue does not require a code change in the dbt repository, or the maintainers are unwilling/unable to merge a Pull Request which implements the behavior described in the issue. |
+| [triage](https://github.com/fishtown-analytics/dbt/labels/triage) | This is a new issue which has not yet been reviewed by a dbt maintainer. This label is removed when a maintainer reviews and responds to the issue. |
```
Comment (drewbanin) [inline CONTRIBUTING.md:41]: :+1:

**Comment by jtcohen6** [comment]
I incorporated feedback from @sanjanasen96 and @aescay. It's still not perfect, but I think this is a step forward.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -97,7 +130,7 @@ brew install postgresql
 
 ### Installation
 
-First make sure that you set up your `virtualenv` as described in section _Setting up an environment_. Next, install dbt (and it's dependencies) with:
+First make sure that you set up your `virtualenv` as described in section _Setting up an environment_. Next, install dbt (and its dependencies) with:
```
Comment (drewbanin) [inline CONTRIBUTING.md:133]: sorry

Comment (jtcohen6) [reply CONTRIBUTING.md:133]: [wasn't you](https://github.com/fishtown-analytics/dbt/commit/69a65dd63f396db3ce3452382cdd37f88ad59539#diff-6a3371457528722a734f3c51d9238c13)

**Pull request merged**